lib/prom/prom.go

```diff
@@ -0,0 +1,121 @@
+package prom
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
+)
+
+//PrometheusMetrics vegeta metrics observer with exposition as Prometheus metrics endpoint
```

Comment:
Please fix the spacing between the // and the start of the comment text everywhere.

Reply:
Please do this.

Reply:
@flaviostutz: Pending.

Reply:
I will work on these on the weekend

Reply:
Whoop whoop! Did you see https://github.com/tsenart/vegeta/pull/534#issuecomment-1630248104 ?

Reply:
done